sqliterepo/sqlx_remote.c

```diff
@@ -34,12 +34,17 @@ License along with this library.
 struct asn_TYPE_descriptor_s;
 
 static MESSAGE
-make_request(const gchar *rn, const struct sqlx_name_s *name, gint64 deadline)
+make_request(const gchar *rn, struct oio_url_s *url,
```

Comment:
With this modification are you sure that it should be merged to 4.x ?

Reply:
If the meta2 is up to date, but the proxy is not up to date, we will receive events of this type:
```
{"event":"storage.container.update","when":1561471129742946,"url":{"ns":"OPENIO"},"request_id":"650AF5C42C0EAD2C7EC0BC0776FF3629","data":{"system":{},"properties":{"test":"1"}}}
```
So during the update, we may have incomplete URLs. But that's not a problem:
- either we need more information and we will fail when updating (no change compared to before)
- or we only need the namespace